bears/verilog/VerilogLintBear.py

```diff
@@ -24,6 +24,7 @@ class VerilogLintBear:
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
     LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Simplification', 'Syntax'}
```

Comment:
when you specify a signal, not sure our bear does that so i did not feel including it?